components/counter/theme.css

```diff
@@ -0,0 +1,35 @@
+@import '@teamleader/ui-colors';
+@import '@teamleader/ui-utilities';
+
+.counter {
+  background: var(--color-ruby);
+  border-radius: 50%;
+  color: var(--color-white);
+  display: inline-block;
+  font-size: calc(1.4 * var(--unit));
+  font-family: var(--font-family-semi-bold);
+  font-weight: 600;
+  line-height: 1;
+}
+
+.medium {
+  min-height: 9px;
+  min-width: 9px;
+}
+
+.small {
+  min-height: 6px;
+  min-width: 6px;
```

Comment:
**PP** Can we use css4 variables here instead? 
Example here: https://github.com/teamleadercrm/teamleader-ui/blob/button-v2/components/button/theme.css